CorsixTH/Lua/config_finder.lua

```diff
@@ -255,6 +256,11 @@ if needs_rewrite then
 'prevent_edge_scrolling = ' .. tostring(config_values.prevent_edge_scrolling) .. '\n' .. [=[
 
 -------------------------------------------------------------------------------------------------------------------------
+-- Capture mouse: By default enabled (capture mouse = true).
+-- ]=] .. '\n' ..
+'capture_mouse = ' .. tostring(config_values.prevent_edge_scrolling) .. '\n' .. [=[
```

Comment:
Too much copy/paste... `config_values.capture_mouse`

Reply:
Maybe mention ctrl-f10 here too. e.g. something like "Initial mouse capture state when the game starts. To toggle in game press ctrl-f10."

Reply:
dang it, I was testing it, and did undo but must have did it on my repo instead of my test folder